lib/spatial_features/has_spatial_features/feature_import.rb

```diff
@@ -68,8 +68,8 @@ def update_spatial_cache(options = {})
 
     def spatial_feature_imports(import_options, make_valid)
       import_options.flat_map do |data_method, importer_name|
-        Array.wrap(send(data_method)).map do |data|
-          spatial_importer_from_name(importer_name).new(data, :make_valid => make_valid) if data.present?
+        Array.wrap(send(data_method)).flat_map do |data|
+          spatial_importer_from_name(importer_name).create(data, :make_valid => make_valid) if data.present?
```

Comment:
This is the core of the change - using `::create` to return 1 or more importer instances